DB/src/main/java/io/deephaven/db/v2/select/SelectColumn.java

```diff
@@ -25,7 +29,7 @@
 public interface SelectColumn {
 
     static SelectColumn of(Selectable selectable) {
-        return selectable.walk(new SelectColumnAdapter()).getOut();
+        return selectable.expression().walk(new ExpressionAdapter(selectable.newColumn())).getOut();
```

Comment:
Could be reduced to `selectable instanceOf ColumnName || selectable.expression() instanceOf ColumnName? new SourceColumn(...) : SelectColumnFactory.getExpression(Strings.of(selectable))`.

Reply:
Ignoring this for now - trying to stay away from instanceOf most of the time.